### ISSUE TITLE
database prefix breaks repositories

### ISSUE THREAD

**Issue by khooz** [GrafiteInc/CMS#128]
Hi,
Here is what I get upon calling `/`:

```
SQLSTATE[42S02]: Base table or view not found: 1146 Table '<dbname>.pages' doesn't exist (SQL: select * from `pages` where `url` = home and `is_published` = 1 and `published_at` <= 2018-03-23 19:27:20 limit 1)
```

I looked at the code and the culprit was the repository
https://github.com/GrafiteInc/CMS/blob/e488e4819237b24fd515f81d866348307aa9bc8e/src/Repositories/PageRepository.php#L24
This should add the database prefix set in the `config/cms.php` as a prefix.
I didn't check other parts of the code and fixed it with not using prefixes at all; but I assume it will greatly help decoupling CMS from rest of the apps.

Cheers!

**Comment by mlantz** [comment]
Thanks, I'll review this today.

**Comment by mlantz** [comment]
Fix will be in the next release

**Comment by mlantz** [comment]
Resolved in v3.1
